Expensify/App
# Update Payments Page to show wallet balance, empty state, and more!

### Details
This PR improves the Payments page and brings it more in line with the original design.
Specifically:
1. Adds the wallet balance
2. Removes the wallet as a payment method to fund the wallet
3. Adds the "Payment Methods" title
4. Adds the empty state for the payment methods list
5. Adds a loading indicator for when we don't have the wallet balance

### Fixed Issues
$ https://github.com/Expensify/Expensify/issues/169463

### Tests / QA
1. Create a new account
2. Go to Settings -> Payments
3. Make sure you see this empty state
![2021-07-06_15-30-41](https://user-images.githubusercontent.com/42391420/124671393-a02eed00-de72-11eb-8cd6-24bfe8df0f42.png)
4. Log in with an account that has bank accounts/cards
5. Make sure you see this state
![2021-07-06_15-51-16](https://user-images.githubusercontent.com/42391420/124671424-acb34580-de72-11eb-9545-6678e05d34e6.png)

### Tested On


### Screenshots
![2021-07-06_16-02-03](https://user-images.githubusercontent.com/42391420/124671979-86da7080-de73-11eb-9c57-265466a94c0b.png)
![2021-07-06_16-01-21](https://user-images.githubusercontent.com/42391420/124671985-880b9d80-de73-11eb-9463-52f82217fd50.png)
